core/controllers/admin.py

```diff
@@ -179,6 +179,10 @@ def post(self):
                     self.payload.get('action') ==
                     'generate_dummy_new_structures_data'):
                 self._load_dummy_new_structures_data()
+            elif (
+                    self.payload.get('action') ==
+                    'generate_dummy_new_skill_data'):
+                self._load_dummy_new_skill()
```

Comment:
I think the name of the method (and the one above) is actually wrong, it should be _generate_dummy_skill_and_questions(), not "load". "Load" implies it is already stored somewhere.

Reply:
Done.